lib/__tests__/_sendEvent.test.ts

```diff
@@ -84,20 +94,189 @@ describe("sendEvent", () => {
     });
     it("should send the correct payload", () => {
       (AlgoliaInsights as any).sendEvent("click", {
-        objectID: "1"
+        eventName: "my-event",
+        indexName: "my-index",
+        objectID: "1",
+        position: 3
       });
       const payload = JSON.parse(sendBeacon.mock.calls[0][1]);
 
       expect(payload).toEqual({
         events: [
           expect.objectContaining({
             eventType: "click",
-            objectID: "1",
-            userID: "42",
+            eventName: "my-event",
+            indexName: "my-index",
+            objectID: ["1"],
+            position: [3],
+            userID: "mock-user-id",
             timestamp: expect.any(Number)
           })
         ]
       });
     });
   });
+
+  describe("eventName", () => {
+    it("should throw if no eventName passed", () => {
+      expect(() => {
+        (AlgoliaInsights as any).sendEvent("click", {
+          indexName: "my-index"
+        });
+      }).toThrowErrorMatchingInlineSnapshot(
+        `"expected required parameter \`eventName\` to be a string"`
+      );
+    });
+    it("should throw if no eventName passed", () => {
```

Comment:
We could use a different title for the the assertion.

Reply:
you mean wrap them in a separate "describe" block?

Reply:
Both of the assertions inside the `describe` have the exact same name. It's easier to follow if they have different name. You don't test the exact same thing in both assertions.

Reply:
my bad, 😅  fixed here: https://github.com/algolia/search-insights.js/pull/33